### ISSUE TITLE
$schema property is incorrectly added to body parameter schemas

### ISSUE THREAD

**Issue by MikeRalphson** [kminami/apib2swagger#23]
Also from http://docs.icons8.apiary.io/api-description-document

The output swagger file contains:

```yaml
  /api/task/web-font/collection:
    post:
      description: ''
      parameters:
        - in: body
          name: body
          schema:
            $schema: 'http://json-schema.org/draft-04/schema#'
            properties:
```

Where `$schema` is not an allowed property of the OpenAPI 2.0 [schema object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schemaObject).

**Comment by kminami** [comment]
Fixed at 0.9.0, thanks!

**Comment by MikeRalphson** [comment]
Many thanks @kminami
